apps/ledger-live-mobile/src/screens/Onboarding/steps/Cards/SelectionCard.tsx

```diff
@@ -0,0 +1,95 @@
+import { Box, Flex, Text } from "@ledgerhq/native-ui";
+import React from "react";
+import { useTheme } from "styled-components/native";
+import { track } from "../../../../analytics";
+import Touchable from "../../../../components/Touchable";
+
+type SelectionCardProps = {
+  title: string;
+  text?: string;
+  icon: React.ReactElement;
+
+  event: string;
+  eventProperties?: Record<string, unknown>;
+
+  onPress: React.ComponentProps<typeof Touchable>["onPress"];
+
+  testID: string;
+};
+
+export const SelectionCards = ({ cards }: { cards: SelectionCardProps[] }) => {
+  const { space } = useTheme();
+
+  return (
+    <>
+      {cards.map((props, index) => (
+        <Box key={index} mb={cards.length - 1 !== index ? space[6] : 0}>
+          <SelectionCard {...props} />
+        </Box>
+      ))}
+    </>
+  );
+};
+
+export const SelectionCard = ({
+  title,
+  text,
+  icon,
+  event,
+  eventProperties,
+  testID,
+  onPress,
+}: SelectionCardProps) => {
+  const { colors, space } = useTheme();
+
+  const pressAndTrack = () => {
+    track(event, eventProperties);
+    onPress?.();
+  };
+
+  return (
+    <Touchable onPress={pressAndTrack} testID={testID}>
+      <Flex
+        flexDirection="row"
+        bg={colors.opacityDefault.c05}
```

Comment:
You don't have to call the useTheme() to get the colors and space, you can directly put bg="opacityDefault.c05" here and py={7} in the Flex and the Box components